app/controllers/application_controller.rb

```
@@ -1,3 +1,2 @@
 class ApplicationController < ActionController::Base
-  protect_from_forgery with: :exception
```

Leave this in the ApplicationController and in the ApiController use `protect_from_forgery with: :null_session`. What this does is check if the session is valid, in the WebApp (Controllers that are going to render HTML and are going to maintain a session) if the session is invalid we want to raise an exception, but in the API ones, since we don't have sessions we use `null_session`